web-components/package.json

```diff
@@ -30,12 +30,11 @@
     "@mapbox/togeojson": "^0.16.0",
     "@math.gl/core": "^4.0.0",
     "@math.gl/web-mercator": "^4.0.0",
-    "@mui/icons-material": "^5.1.0",
     "@mui/lab": "^5.0.0-alpha.54",
-    "@mui/material": "^5.10.10",
-    "@mui/styled-engine": "^5.10.8",
-    "@mui/styles": "^5.10.10",
-    "@mui/system": "^5.10.10",
+    "@mui/material": "^5.16.12",
```

Comment:
this was to fix this https://github.com/mui/material-ui/issues/41388#issuecomment-2506157151